_data/vangelists/vangelists.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Vangelis Tsiatsianas
+institution: Computer Science Department - University of Crete
+profile_pic: vangelists.png
+quote: Bazinga! 💩
```

Comment:
Remove the 💩 (poop emoji) as it is a non-ascii character.

Reply:
Hi Mohammed,

@juanpflores has mentioned that emojis are allowed: https://github.com/education/graduation/issues/348#issuecomment-631776551.